Jenkinsfile

```diff
@@ -77,11 +77,6 @@ node ('master') {
             sh './bin/run_tests'
         }
 
-        stage ("Build documentation") {
-            sh 'docker build . -f sawtooth-build-docs -t sawtooth-build-docs:$ISOLATION_ID'
-            sh 'docker run --rm -v $(pwd):/project/sawtooth-seth sawtooth-build-docs:$ISOLATION_ID'
-        }
-
```

Comment:
Is there a Jira ticket to read docs build?

Reply:
I already have a Docs PR that re-adds this